### PR TITLE
0.10.0 version bump with CI debugging

### PR DESCRIPTION
**Pull request by Shoeboxam**
To fix the CI failure, I just removed the cache.

### CODE REVIEW

**Comment by Shoeboxam** [comment]
* **#1687** <a href="https://app.graphite.dev/github/pr/opendp/opendp/1687?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @Shoeboxam and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by mccalluc** [comment]
Similar erratic R error: https://github.com/opendp/opendp/actions/runs/9565406494/job/26368902658
```
── 1. Error ('test-measurements.R:64:3'): test_gaussian_curve ──────────────────
Error: Error in `make_zCDP_to_approxDP(then_gaussian(input_space, 4))`: [FFI] : null pointer: measurement
Error: Error: Failures detected.
Execution halted
disabled backtrace
Backtrace:
    ▆
 1. └─opendp::make_zCDP_to_approxDP(then_gaussian(input_space, 4)) at test-measurements.R:64:3
 ```
Restarted, and will add link to existing issue.

File: CHANGELOG.md [hunk 1]
```diff
@@ -153,7 +154,7 @@ showing the source changes from the previous version.
     - Error on warning from devtools::check [#1253](https://github.com/opendp/opendp/pull/1253)
 
 
-## [0.9.2-dev](https://github.com/opendp/opendp/compare/v0.9.1...HEAD) - TBD
+## [0.9.2](https://github.com/opendp/opendp/compare/v0.9.1...v0.9.2) - 2024-02-08
```
Comment (mccalluc) [inline CHANGELOG.md:157]: I don't this this tag was made: the link doesn't really work: https://github.com/opendp/opendp/compare/v0.9.1...v0.9.2

- See #1628

File: python/setup.cfg [hunk 2]
```diff
@@ -35,22 +35,17 @@ opendp =
...
 numpy = 
 	numpy>=1.17,<2.0
 	randomgen
 scikit-learn = 
 	scikit-learn
 	%(numpy)s
-polars =
-	# A strict version requirement is necessary to keep the serialization format of LazyFrames stable.
-	# If you change the version of Polars, 
-	# be sure to also change the Polars version in rust/Cargo.toml and test binary compatibility.
```
Comment (mccalluc) [inline python/setup.cfg:48]: Is there a reason to delete the comments? My sense is that these are still true, and they point out changes that aren't obvious and would need to be made across multiple files.

**Review by mccalluc — COMMENTED** [review]
Restore comments? but if they really aren't needed any more, I should understand the big picture better.

File: python/setup.cfg (hunk 2 above)
Comment (Shoeboxam) [reply python/setup.cfg:48]: I see what happened: these comments are not preserved by ConfigParser. I swapped to another package ConfigUpdater.

**Review by mccalluc — APPROVED** [review]
Looks good. Approved! (Might be good for us to do a patch release in a few weeks and check if Raman or I can follow the process.)

**Pull request merged**